helm/chart/values.yaml

```diff
@@ -4,7 +4,8 @@
 
 replicaCount: 1
 
-deployableNamespaces: []
+deployableNamespaces:
+  - whoami
```

Comment:
Shouldn't we keep this as is and only overwrite using value files or the skaffold file?

Reply:
We have [the `whoami` group as a default value in the chart for the `im-manager`](https://github.com/dhis2-sre/im-manager/blob/master/helm/chart/values.yaml#L12-L14), so I thought it might be nice to keep it consistent for the inspector, too.